### PR TITLE
feat: Add alert page 

### PR DESCRIPTION
**Pull request by yokwejuste**
Issue #117 

Alert page added together with the route and different required items.

### CODE REVIEW

**Comment by Asam237** [comment]
@yokwejuste 
Screenshot from 2022-08-10 08-17-16

**Comment by Sanix-Darker** [comment]
Did you tested this pull request ? @yokwejuste ??

**Review by Sanix-Darker — CHANGES_REQUESTED** [review]
Did you tested this pull request ? @yokwejuste ??

File: frontend/public/index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
-  <title>React App</title>
+  <title>ONACUT Project</title>
```
Comment (Asam237) [inline frontend/public/index.html:15]: This is not the right project name, just put <Onacut | Add alert>.

File: frontend/src/assets/data/Items.data.ts [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
+        name: LANGUAGE.home.info,
+        link: "/conseils"
+    },
+    {
+        icon: listes,
+        name: LANGUAGE.home.vue,
+        link: "/lists"
+    },
+    {
+        icon: report,
```
Comment (Asam237) [inline frontend/src/assets/data/Items.data.ts:25]: Add a .png icon as this one does not give

sp01

File: frontend/public/index.html (hunk 1 above)
Comment (yokwejuste) [reply frontend/public/index.html:15]: This is the general name for the project not the page name.

File: frontend/src/pages/addAlert/addAlert.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
+import React, {useState, useEffect} from "react";
+import './addAlert.css';
```
[the remark this answers is not in the document]
Comment (yokwejuste) [reply frontend/src/pages/addAlert/addAlert.tsx:2]: Should i remove the CSS file?

File: frontend/src/pages/home/Home.pages.tsx [hunk 4]
```diff
@@ -58,7 +58,9 @@ const Home = () => {
                                     alt=""
                                     className="w-6 h-6 mr-2"
                                 />
-                                {LANGUAGE.home.signalerPanne}
+                                <Link to={'/alert'}>
```
[the remark this answers is not in the document]
Comment (yokwejuste) [reply frontend/src/pages/home/Home.pages.tsx:61]: Did you finally check the update ?

**Pull request merged**